src/shared-components/DatePicker/DatePicker.tsx

```
@@ -17,11 +25,67 @@ import {
   DatePicker as MuiDatePicker,
   LocalizationProvider,
 } from '@mui/x-date-pickers';
+import type { Dayjs } from 'dayjs';
+import styles from './DatePicker.module.css';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { FormFieldGroup } from '../FormFieldGroup/FormFieldGroup';
 import styles from './DatePicker.module.css';
```

Biome (2.3.13)

[error] Shouldn't redeclare 'styles'. Consider to delete it or rename it.

'styles' is defined here:


(lint/suspicious/noRedeclare)